### PR TITLE
Updates to the README

### PR DESCRIPTION
**Pull request by adil-hussain-84**
- Updated the Requirements section in the README to specify clearly the versions of Gradle and J2ObjC that are supported.
- Updated the version of the J2ObjC Gradle plugin in the README to the latest version.

### CODE REVIEW

**Comment by adil-hussain-84** [comment]
@brunobowden recreated pull request as a single commit.

File: README.md [hunk 1]
```diff
@@ -81,13 +81,13 @@ For more complex situations like:
...
 
-### Minimum Requirements
+### Requirements
 
 The plugin requries modern versions of Gradle and J2ObjC, and assumes the
 [J2ObjC Requirements](http://j2objc.org/#requirements):
 
-* Gradle 2.4
-* J2ObjC 0.9.8.2.1 or higher
+* Gradle 2.4 to 2.8 supported
```
Comment (brunobowden) [inline README.md]: Add mention of the Gradle 2.9 incompatibility:

    Gradle 2.4 to 2.8 supported
    (Gradle 2.9+ is unsupported - see [issue 568](https://github.com/j2objc-contrib/j2objc-gradle/issues/568))

**Comment by brunobowden** [comment]
Please see this notice about the future of the J2ObjC Gradle Plugin:
https://groups.google.com/forum/#!topic/j2objc-discuss/jQ_eyW9nxmQ

**Comment by brunobowden** [comment]
Closing pull request as the plugin is no longer being actively maintained. You're welcome to fork the project.

**Pull request closed without merging**